index.html

```diff
@@ -20,43 +23,60 @@
-
   </head>
   <body data-spy="scroll" data-target="#navSidebar" data-offset="50">
-
     <nav class="navbar navbar-fixed-top navbar-light navbar-color">
       <div class="container-fluid">
         <div class="navbar-header">
-          <button type="button" class="navbar-toggle collapsed navbar-collapse-button" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+          <button type="button"
+                  class="navbar-toggle collapsed navbar-collapse-button"
+                  data-toggle="collapse" data-target="#navbar"
+                  aria-expanded="false" aria-controls="navbar">
             <span class="sr-only">Toggle navigation</span>
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
           </button>
-          	<div class="block">
-	          <div class="menubtn-and-brand">
-	            <button type="button" class="btn menu-btn" data-toggle="collapse" data-target="#navSidebarMenu"><span class="glyphicon glyphicon-th"></span></button>
-	            <a class="navbar-brand brand" href="http://apps.loklak.org">
-	            loklak <span>apps</span>
-	          </a>
-	          </div>
-	          <div class="form-group search">
-	            <input type="text" class="form-control" id="usr" ng-model="searchTerm"
-	            ng-init="searchTerm = ''" placeholder="Search apps by app name or author's name">
-	            <span class="input-group-btn">
-	              <button class="btn loklak-btn" type="submit"><span class="glyphicon glyphicon-search"></span></button></span>
-	          </div>
-	        </div> 
+<<<<<<< HEAD
+            <div class="block">
+            <div class="menubtn-and-brand">
+              <button type="button" class="btn menu-btn"
+                      data-toggle="collapse"
+                      data-target="#navSidebarMenu">
+                      <span class="glyphicon glyphicon-th"></span>
+              </button>
+              <a class="navbar-brand brand" href="http://apps.loklak.org">
+              loklak <span>apps</span>
+            </a>
+            </div>
+            <div class="form-group search">
+              <input type="text" class="form-control" id="usr" ng-model="searchTerm"
+              ng-init="searchTerm = ''" placeholder="Search apps by app name or author's name">
+              <span class="input-group-btn">
+                <button class="btn loklak-btn" type="submit">
+                  <span class="glyphicon glyphicon-search"></span>
+                </button>
+              </span>
+            </div>
+          </div>
+=======
+          <a class="navbar-brand brand" href="//apps.loklak.org">
+          loklak <span>apps</span>
+          </a>
+          <div class="form-group search"><input type="text" class="form-control" id="usr" ng-model="searchTerm" ng-init="searchTerm=''" placeholder="Search apps by app name or author"s name"></div>
```

Comment:
Line is longer than allowed. (197 > 129)

*LineLengthBear, severity NORMAL, section `default`.*